thoth/common/openshift.py

```diff
@@ -1526,6 +1527,7 @@ def schedule_mi_workflow(
             "ENTITIES": entities,
             "KNOWLEDGE_PATH": knowledge_path,
             "MI_THOTH": "1" if mi_used_for_thoth else "0",
+            "MI_MERGE": "1" if mi_merge else "0",
```

Comment:
Hint: booleans are ints in python: `str(int(mi_merge))`

Reply:
I know, I was just following other declarations in this file as well, see https://github.com/thoth-station/common/blob/5b96cd3c2ddf4856426f69f87421215643f3f4cc/thoth/common/openshift.py#L1368

Can I then refactor these as e.g. `"MI_MERGE": mi_merge"`

Reply:
> Hint: booleans are ints in python: `str(int(mi_merge))`

ooh I get it now, you mean it's better to use casting than the ternary operator